wdl/Utils.wdl

```diff
@@ -313,3 +313,47 @@ task SubsetPedFile {
     maxRetries: select_first([runtime_attr.max_retries, default_attr.max_retries])
   }
 }
+
+task LocalizeCloudFileWithCredentials {
+  input {
+    String cloud_file_path
+    String service_account_json
+    Int disk_size
+    String cloud_sdk_docker
+    RuntimeAttr? runtime_attr_override
+  }
+
+  RuntimeAttr default_attr = object {
+    cpu_cores: 1,
+    mem_gb: 3.75,
```

Comment:
Is 3.75 optimal? Can you use 0.9?

Reply:
Just tested and LocalizeCram completed successfully with 0.9GB memory on a new sample with a similar runtime, so seems fine to set that as default